### PR TITLE
Travis ci

### PR DESCRIPTION
**Pull request by ewilz**
A few things going on here:

- Took `src/tests/build` out of `.gitignore` so the built contracts are included with repo. Setting up an entire node/truffle environment for test contracts that are very unlikely to change much seemed like a lot of overhead for situations with no node/truffle. 

- `.travis.yml` split into 2 jobs. no node/truffle.

- `./build.sh` no longer fails silently

- Updated rust-toolchain

### CODE REVIEW

File: src/tests/mod.rs [hunk 1]
```diff
@@ -135,7 +135,7 @@ fn test_contract_to_eoa_transfer() {
 fn test_get_code() {
     test_utils::run_test(0, |contract| {
         let test_addr = contract.deploy_code(TEST.to_string());
-        assert!(contract.get_code(test_addr).len() > 3000);
+        assert!(contract.get_code(test_addr).len() > 3001); // contract code should roughly be over length 3000
```
Comment (ewilz) [inline src/tests/mod.rs]: toggled to test build, I can make it 3000 for official mergeable PR

File: src/tests/build.sh [hunk 2]
```diff
@@ -6,18 +6,20 @@ cat build/contracts/SolTests.json | \
...
   tr -d '\n' \
-  > build/subcontract.bin
+  > build/SubContract.bin
 
 cat build/contracts/SubContract.json | \
   jq .abi \
-  > build/subcontract.abi
+  > build/SubContract.abi
+
+rm -rf build/contracts
```
Comment (ewilz) [inline src/tests/build.sh:25]: built then unused.

**Comment by prestwich** [comment]
> . is ./res/near_evm.wasm necessary to properly do the lib tests?

No it is not. We still want to leave in the build step so we can be sure we don't introduce changes that break the wasm build, but it can be paralellized

File: src/tests/mod.rs (hunk 1 above)
Comment (ewilz) [reply src/tests/mod.rs]: oh wait i don't have to do this anymore!! This was a docker thing 😂

**Review by prestwich — APPROVED** [review]

**Pull request merged**